src/stream/src/executor/group_top_n.rs

```diff
@@ -204,7 +204,7 @@ impl<S: StateStore> TopNExecutorBase for InnerGroupTopNExecutorNew<S> {
                 )
                 .await?;
         }
-        // compare the those two ranges and emit the differantial result
```

Comment:
The comment is confusing

Reply:
maybe it should be the comment of the function `generate_output`